parlai/core/torch_ranker_agent.py

```diff
@@ -387,14 +387,11 @@ def _set_label_cands_vec(self, *args, **kwargs):
         Useful to override to change vectorization behavior.
         """
         obs = args[0]
-        cands_key = (
-            'candidates'
-            if 'labels' in obs
-            else 'eval_candidates'
-            if 'eval_labels' in obs
-            else None
-        )
-        if cands_key is not None and self.opt[cands_key] not in [
+        if 'labels' in obs:
```

Comment:
can you explain this change? is there an instance where 'labels' and 'eval_labels' don't exist but 'label_candidates' do?

Reply:
discussed offline: during interactive chat you dont have a label but do have fixed cands